lib/web/css/source/lib/_resets.less

```diff
@@ -49,9 +49,10 @@
 
     img {
         border: 0;
+        height: auto;
+        max-width: 100%;
```

Comment:
Great! I believe your fix make https://github.com/magento/magento2/pull/25623/files obsolete now. Could you remove the changes that PR introduced and verify that all images including logo work fine?

Reply:
Hello @krzksz,

I have updated changes as requested. 

Thanks :)